java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -3844,6 +3821,34 @@ public void zincrby_returns_success() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
```

Comment:
Is is just moved or something changed?

Reply:
Just moved... to put the xadd tests near the remainder of the stream tests.  

No changes.